internal/telemetry/client.go

```diff
@@ -174,24 +172,22 @@ func (c *client) Start(configuration []Configuration) {
 	c.metrics = make(map[Namespace]map[string]*metric)
 	c.debug = internal.BoolEnv("DD_INSTRUMENTATION_TELEMETRY_DEBUG", false)
 
+	productInfo := Products{
+		AppSec: ProductDetails{
+			Version: version.Tag,
+			Enabled: appsec.Enabled(),
+		},
+	}
+	productInfo.Profiler = ProductDetails{
+		Version: version.Tag,
+		// if the profiler is the one starting the telemetry client,
```

Comment:
just to clarify, and if it's not the one starting the telemetry client then this is OK since the profiler will send a separate "product change" event ?

Reply:
Yep!